README.md

```diff
@@ -1,67 +1,76 @@
-# SetReplace
+# Set Substitution System
 
-## Set | Network Substitution Systems
+## Basic Example
 
-**SetReplace** is a Wolfram Language package that implements a substitution system such that in each step an unordered subset matching a given pattern is deleted from a multiset and replaced with different subset. If each element of the set consists of pairs of elements, this set can be thought of as a directed graph, and the system becomes a ***network substitution*** (aka *graph rewrite*) ***system***.
-
-An example of such a system would be removing vertices from a path graph
+**SetReplace** is a [Wolfram Language](https://www.wolfram.com/language/) package to manipulate set substitution systems. To understand what a set substitution system does consider an unordered set of elements:
 ```
-In[.] := SetReplace[{{0, 1}, {1, 2}, {2, 3}}, {{a_, b_}, {b_, c_}} :> {{a, c}}, 2]
+{1, 2, 5, 3, 6}
 ```
+We can set up an operation on this set which would take any of the two elements and replace them with their sum:
 ```
-Out[.] = {{0, 3}}
+{a_, b_} :> {a + b}
 ```
-
-Here, `{{0, 1}, {1, 2}, {2, 3}}` is the initial condition, `{{a_, b_}, {b_, c_}} :> {{a, c}}` is the rule that matches pairs of edges sharing a vertex and replaces them with a single edge, and `2` the number of steps, i.e., the number of times the rule is applied.
-
-One can also imagine an inverse of the rule above
+In **SetReplace**, this can be expressed as (the new element is put at the end)
 ```
-In[.] := SetReplace[{{0, 1}}, {{a_, b_}} :>
-  Module[{new}, {{a, new}, {new, b}}], 2]
+In[] := SetReplace[{1, 2, 5, 3, 6}, {a_, b_} :> {a + b}]
+Out[] = {5, 3, 6, 3}
```

Comment:
At some point, you had a note that 1+2 = 3 is the last element of the list.. I think that note was very helpful

Reply:
The note is still there, just before this input. I also now explicitly say that the new element is `1 + 2 -> 3`.